server/documents/modules/dropdown.html.eco

```diff
@@ -1681,6 +1681,43 @@ themes      : ['Default', 'GitHub', 'Material']
       </div>
     </div>
 
+    <div class="tag example">
+      <h4 class="ui header">Stuck User Additions</h4>
+      <p>While using <code>allowAdditions: true</code>, the first line indicating "add:" with the user addition may not be correctly displayed, depending on the container size and the distance between topmost visible item and menu's top item.
```

Comment:
- Change from tag to stuck
- Please add the usual black label stating that this feature is new in 2.9.0
- You forget the necessary setting `hideAdditions:false`
```suggestion
    <div class="stuck example">
      <h4 class="ui header">Stuck User Additions <span class="ui black label">New in in 2.9.0</span>/h4>
      <p>While using <code>allowAdditions: true</code> together with <code>hideAdditions: false</code>, the first line indicating "add:" with the user addition may not be correctly displayed, depending on the container size and the distance between topmost visible item and menu's top item.
```